resources/views/world/_feature_entry.blade.php

```diff
@@ -7,7 +7,7 @@
         @if($feature->feature_category_id)
             <div><strong>Category:</strong> {!! $feature->category->displayName !!}</div>
         @endif
-        @if($feature->parent_id)
+        @if($feature->parent_id && $feature->parentVisible)
```

Comment:
Switch this to `@if($feature->parent_id && $feature->parent->is_visible)`

Reply:
Edited, tested- worked!